packages/data-point/lib/reducer-types/reducer-helpers/reducer-omit/factory.test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env jest */
+'use strict'
```

Comment:
Do we need to have these `'use strict'` lines?

Reply:
No, this is just existing code. We should open an issue to remove `'use strict'` everywhere.

Reply:
#131